extensions/wikia/ARecoveryEngine/js/InstartLogic/recovery.js

```diff
@@ -1,8 +1,10 @@
+/*global define*/
 define('ext.wikia.aRecoveryEngine.instartLogic.recovery', [
 	'ext.wikia.adEngine.adContext',
+	'ext.wikia.adEngine.provider.gpt.targeting',
 	'wikia.log',
 	'wikia.window'
-], function (adContext, log, win) {
+], function (adContext, targeting, log, win) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "win". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)